### PR TITLE
Chris/create preferences page

### PR DESCRIPTION
**Pull request by ctlchan**
Made a route for the preferences page which now contains all three components. Adjusted the time component(s) slightly to make it work as a single page. 

If you try it, you can check the console for the stored preferences after you press the "save preferences" button. Formatting for preferences has not necessarily been addressed yet.

### CODE REVIEW

File: web/meetup-facilitator/src/App.jsx [hunk 1]
```diff
@@ -16,12 +17,7 @@ function App() {
           <Route path='/' element={<Home/>} />
           <Route path='/activities' element={<Activities/>} />
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/App.jsx]: Let's also remove the `/activities` route

File: web/meetup-facilitator/src/components/LocationComponent.jsx [hunk 2]
```diff
@@ -26,18 +27,23 @@ function LocationComponent() {
...
+    prefsContext.updateLocationPrefs(newCoordinates, sliderValue);
   }
 
   const handleSliderChange = (event, newValue) => {
     setSliderValue(newValue);
+    prefsContext.updateLocationPrefs(coordinates, newValue);
+
   };
 
   return (
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/components/LocationComponent.jsx:44]: Let's remove
```
<h1>Place Google Maps API here</h1>
```

File: web/meetup-facilitator/src/components/TimeMain.jsx [hunk 3]
```diff
@@ -1,42 +1,56 @@
...
   ]);
+  const [showHours, setShowHours] = useState(false);
 
-  const handleGoToTimeHour = () => {
+  const handleSelectTime= () => {
     if (dates.length > 0) {
-      navigate(`/TimeHour?dates=${JSON.stringify(dates)}`);
+      // navigate(`/TimeHour?dates=${JSON.stringify(dates)}`);
+      dates.map((date) => {
+      } )
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/components/TimeMain.jsx]: Don't think there's a need for this

File: web/meetup-facilitator/src/components/LocationComponent.jsx (hunk 2 above)
Comment (pabloj2001) [reply web/meetup-facilitator/src/components/LocationComponent.jsx:44]: Can we also only show the `GoogleMapComponent` if an `address` has been selected?

File: web/meetup-facilitator/src/components/TimeHour.jsx [hunk 4]
```diff
@@ -64,18 +66,19 @@ export default function TimeHour() {
...
 
     setFormattedDates(formattedSelection);
+    prefsContext.updateTimePrefs(formattedSelection);
   };
 
   const handleJumpToLocationPage = () => {
     // Use the navigate function to go to the LocationPage
-    navigate("/LocationPage");
+    navigate("/LocationPage");                                                                                                              
   };
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/components/TimeHour.jsx]: `handleJumpToLocationPage` No longer necessary

File: web/meetup-facilitator/src/components/TimeHour.jsx [hunk 5]
```diff
@@ -1,11 +1,13 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useContext } from "react";
 import { useLocation, Link, useNavigate } from "react-router-dom"; // Import Link and useNavigate from react-router-dom
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/components/TimeHour.jsx]: No longer needed

File: web/meetup-facilitator/src/components/TimeMain.jsx [hunk 6]
```diff
@@ -1,42 +1,56 @@
 import React, { useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/components/TimeMain.jsx]: No longer needed

**Review by pabloj2001 — COMMENTED** [review]
Good job merging the pages! A few minor things to update. Like you mentioned, let's move on to something else and we'll take up connecting to the BE later

**Review by pabloj2001 — APPROVED** [review]
LGTM

**Review by marissa-anj — APPROVED** [review]
love the formatting gj

**Pull request merged**